README.md

```diff
@@ -3,7 +3,9 @@
 # EchoCore
 
 A monorepo with all tools needed to communicate with echo framework. the repo contains two packages.
-You can read more about them in the links below, which also includes breaking changes (new since v0.5.0).
+You can read more about them in the links below.
+
+Also see **breaking changes** (new since v0.5.0):
```

Comment:
Totally subjective: maybe could be a bit more expressive. (for me it's not clear what "also see" is referring to)
Maybe something like this below?
--
Added new readme section: **breaking changes**. Be sure to check it out!